{
    "errors": [],
    "fails": [],
    "findings": [
        {
            "column": "10",
            "column_end": "42",
            "filename": "dataset/reentrancy/reentrance.sol",
            "line": "24",
            "line_end": "24",
            "name": "Reentrancy Vulnerability"
        },
        {
            "column": "5",
            "column_end": "33",
            "filename": "dataset/reentrancy/reentrance.sol",
            "line": "14",
            "line_end": "14",
            "name": "Result of expression can be over- or under-flown by external entity"
        }
    ],
    "infos": [],
    "parser": {
        "id": "ccc",
        "mode": "solidity",
        "version": ""
    }
}
